Adjust the set of standard locations that the erlang erts configure script
uses to try to find SSL. Without this change, we see the following error
at pkglint time:

ERROR userland.action001.3        bad RUNPATH, 'usr/lib/amd64/erlang/lib/crypto-3.5/priv/lib/crypto.so' includes '/usr/local/lib/64:/opt/local/lib/64:/usr/pkg/lib/64:/usr/local/openssl/lib/64:/usr/lib/openssl/lib/64:/usr/openssl/lib/64:/usr/local/ssl/lib/64:/usr/lib/ssl/lib/64:/usr/ssl/lib/64://lib/64:/usr/local/lib:/opt/local/lib:/usr/pkg/lib:/usr/local/openssl/lib:/usr/lib/openssl/lib:/usr/openssl/lib:/usr/local/ssl/lib:/usr/lib/ssl/lib:/usr/ssl/lib://lib'

This change will not be sent upstream.

diff -r 395866bff381 lib/crypto/configure.in
--- a/lib/crypto/configure.ac	Fri May 28 15:48:39 2021 +0000
+++ b/lib/crypto/configure.ac	Fri May 28 15:51:09 2021 +0000
@@ -77,7 +77,7 @@
 
 AC_CHECK_SIZEOF(void *)
 
-std_ssl_locations="/usr/local /usr/sfw /usr /opt/local /usr/pkg /usr/local/openssl /usr/local/opt/openssl /usr/lib/openssl /usr/openssl /usr/local/ssl /usr/lib/ssl /usr/ssl /"
+std_ssl_locations="/usr"
 			
 AC_ARG_WITH(ssl,
 AS_HELP_STRING([--with-ssl=PATH], [base location of OpenSSL include and lib directories])
